.github/workflows/ci.yml

```diff
@@ -71,13 +70,26 @@ jobs:
       uses: py-actions/py-dependency-install@v2
       with:
         path: requirements.txt
+    - name: Install Itself
```

Comment:
FYI this is grammatically incorrect — saying that pip installs itself means that pip installs pip. Maybe it would be acceptable to say "Self-install" but even then, it would mean that the project installs itself (as in aiohttp installs aiohttp).
A better way could be to say "Install the tested project to the current environment in the editable mode"

Reply:
Feel free to fix please